Automattic/wp-calypso
# Users management: Update users profiles

#### Proposed Changes

* User details screens adjustments
  * Added formatted header above the user details screens
  * Added CTA buttons (moved from list item)
  * Adjusted team member details page (replace radio buttons with dropdown)
  * Created subscriber user details screen with a new dedicated route `/people/subscribers/{site_slug}/{id}`
* List item adjustments
  * Adjusted badge background colors
  * Got rid of the CTA button from list the list item component
* Team tab content adjustments
  * Created accepted invites block and placed it below the pending
  * Hide pending and accepted invites if there are no invites

Design proposal: pbAok1-3rs-p2

Some stuff is hidden behind the feature flag; if you want to test it with the direct link, provide the flag with query param: `?flags=user-management-revamp`

#### Testing Instructions

Subscribers tab:
* Go to `/people/subscribers/{site_slug}`
* Check the design of the list item components
* Check if there are label distinctions between the `Follower` and `Email subscriber`
* Select on the item
* Check the User details screen (it's the new component)

Team tab:
* Click on the `Team` tab
* Add team members
* Check pending invites block
* Accept the invite you made
* Check the accepted invites block

#### Screenshots

<table>
<tr>
<td width="50%"><strong>Before:</strong></td>
<td width="50%"><strong>After:</strong></td>
</tr>
<tr>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209696514-80f874d6-8b90-4369-9bac-e742122a4b2d.png" />
</td>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209696554-f5180944-ea85-4ea3-8791-48d6bb8ed6ce.png" />
</td>
</tr>

<tr>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209697022-08f71888-6252-49d4-8b22-028e3e7639ff.png" />
</td>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209697115-e7a18189-ec90-4955-9460-95a9b8e63ea3.png" />
</td>
</tr>

<tr>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209725453-84964801-932c-4481-abcc-1cf44e43ef6d.png" />

</td>
<td width="50%">
<img src="https://user-images.githubusercontent.com/1241413/209725465-63ce8169-e84c-4018-a69b-e1b90af50be0.png" />
</td>
</tr>

<tr><td colspan="2">
<img width="745" alt="Screenshot 2022-12-30 at 10 46 11" src="https://user-images.githubusercontent.com/1241413/210072681-27c231b8-9452-4d1f-b2e3-aefeb2b7dc82.png">
</td></tr>


<tr>
<td width="50%"></td>
<td width="50%"></td>
</tr>

</table>

#### Pre-merge Checklist





Related to #70698
